utils/build-common.sh

```diff
@@ -113,8 +120,18 @@ function building_electron_app {
 }
 
 function macos_code_sign {
+    # prerequisites for this:
+    # in short:
+    # * make sure you have a proper app-specific password on https://appleid.apple.com/account/manage
+    # * collect some information via scrun altool --list-providers -u "<yourAppleID>"
```

Comment:
`scrun` is definitely a type. Also, aren't transitioning away from `altool` in this PR? In any case, the command was not available for me.

Reply:
True. However the information (TeamID) can also be found somewhere else.